src/plugins/loan-channel-query-map.js

```diff
@@ -9,25 +11,29 @@ export default {
 					id: 5,
 					url: 'women',
 					queryParams: 'status=fundRaising&gender=female&riskRating=0,5',
-					algoliaParams: 'gender=female'
+					algoliaParams: 'gender=female',
+					flssLoanSearch: { gender: 'female' },
 				},
 				{
 					id: 8,
 					url: 'agriculture',
 					queryParams: 'status=fundRaising&riskRating=0,5&sector=1',
-					algoliaParams: 'sector=Agriculture'
+					algoliaParams: 'sector=Agriculture',
+					flssLoanSearch: { sector: [1] },
```

Comment:
I think we need a key of `sectorId` when we're using the `id`.

Reply:
Oops, good catch. Updated!